### PR TITLE
Design + mock data for waypoint/node tables

### LINKED ISSUE #256: Choose table UI library

### LINKED ISSUE #257: Create generic table layout

### LINKED ISSUE #258: Create waypoint table

### LINKED ISSUE #259: Create node table

### PR DESCRIPTION
**Pull request by samfydega**
This PR creates the UI for the 'Manage Waypoints' and 'Manage Nodes' pages in the client, specifically the tables that will be populated for reviewing and filtering waypoint/node data. The pages are not currently functional. The HeaderComponent is the specific in-line styling for table elements to be implemented in the next PR.

**Node page**
c1af0d419ed0d603060b00d4685be773

**Waypoint page**
d098f44b6e6c88f263051b2bcbef94e6-1

Closes #256 
Closes #257 
Closes #258 
Closes #259

### CODE REVIEW

**Comment by ajmcquilkin** [comment]
Would you mind including a screenshot of the changes?

File: package.json [hunk 1]
```diff
@@ -29,6 +31,7 @@
     "react-map-gl": "^7.0.19",
     "react-redux": "^8.0.4",
     "react-router-dom": "^6.6.1",
+    "react-table": "^7.8.0",
```
Comment (ajmcquilkin) [inline package.json:34]: Why do we need both this and `@tanstack/react-table`? Also, why does this PR add in `mapbox-gl` as a dependency?

Comment (samfydega) [reply package.json:34]: We can remove "react-table": "^7.8.0" and keep the tanstack version. The mapbox-gl was a dependency from tanstack I believe.

Comment (samfydega) [reply package.json:34]: Also added screenshots

**Comment by ajmcquilkin** [comment]
Updated to the following UI pattern:
image

File: src/components/pages/ManageNodePage.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      {
+        id: "Longitude",
+        accessorFn: (n) => n.position?.longitudeI ?? "No GPS lock",
+      },
+      {
+        id: "Battery",
+        accessorFn: (n) =>
+          n.deviceMetrics?.batteryLevel && n.deviceMetrics.batteryLevel
+            ? `${n.deviceMetrics.voltage}V (${
+                n.deviceMetrics.batteryLevel * 100
```
Comment (sherry1iu) [inline src/components/pages/ManageNodePage.tsx:35]: Not blocking, but I don't completely understand the calculation here and I don't think it's right:

image

**Review by sherry1iu — APPROVED** [review]
Looks good! Just a few small things, but not blocking - I think it might be a problem with firmware.

File: src/components/pages/ManageNodePage.tsx (hunk 2 above)
Comment (sherry1iu) [reply src/components/pages/ManageNodePage.tsx:35]: Similarly, is location being set as a default of 0?

Comment (ajmcquilkin) [reply src/components/pages/ManageNodePage.tsx:35]: Hmm interesting, haven't seen that one! Might have multiplied by one too many scaling factors on the percentage

Comment (ajmcquilkin) [reply src/components/pages/ManageNodePage.tsx]: @sherry1iu Should be fixed! There was an extra 100x multiplier

**Review by uhuruhashimoto — APPROVED** [review]
Hardware tested with two nodes; correctly displaying all info. Note: there's currently no user interaction with the tables (MVP).

**Pull request merged**